cfgov/v1/jinja2/v1/includes/molecules/translation-links.html

```diff
@@ -2,11 +2,13 @@
 
     Render a list of translation links, given:
 
-    value.language: The current page language code, e.g. "en".
+    language: The current page language code, e.g. "en".
+              We already should have {{ language }},
+              because it was passed by context.
 
-    value.links:    A list of translation links.
+    links: A list of translation links.
```

Comment:
It'd be nice to document that each of these should have a language, text, and href attribute.

Reply:
It does below it in the comment block

Reply:
D'oh, so it does!